packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -152,14 +159,6 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
     return new RichTextToolbarPlugin();
   }, []);
 
-  const isDarkThemedValue = useMemo(() => {
-    return isDarkThemed(theme);
-  }, [theme]);
-
-  useEffect(() => {
-    editor.current?.setDarkModeState(isDarkThemedValue);
```

Comment:
we don't need this anymore as editor creation `useEffect` already depends on theme and will be updated when theme is changed